### ISSUE TITLE
Sortable does not navigate through items when screen reader is on

### ISSUE THREAD

**Issue by filipKovachev** [telerik/kendo-react#1322]
When having a screen reader turned on, the Sortable component cannot navigate through items. 

Can be reproduced here: https://stackblitz.com/edit/react-mfzrhy?file=app/main.jsx

Ticket ID: 1567285

**Comment by vveesseelliinnaa** [comment]
A keyboard combination that works with NVDA (probably a combination with the [tool's built-in shortcuts](https://dequeuniversity.com/screenreaders/nvda-keyboard-shortcuts#nvda-nvda_shortcut_keys)):
- Shift + Alt + Arrow (or Alt + Arrow) - moves the outline
- Shift + Alt + Ctrl + Arrow - moves the item

**Comment by iadnonov** [comment]
This behavior is not reproducible in version 8.2.0-develop.37.
